integration-tests/quartz/src/main/resources/application.properties

```diff
@@ -18,3 +18,6 @@
 camel.component.quartzFromProperties.propertiesFile = quartz.properties
 
 quarkus.native.resources.includes = quartz.properties
+
+#quick workaround caused by https://issues.apache.org/jira/browse/CAMEL-18143
```

Comment:
Is this an isssue ?

Reply:
There is a PR to fix that. But it depends on something that'll be available in Quarkus 2.14:

https://github.com/apache/camel-quarkus/pull/4153

Reply:
Many thanks! I wanted to make sure there's an open issue